src/resources/elements/primeDesignSystem/demos/demos.ts

```diff
@@ -35,6 +35,20 @@ export class Demos {
         name: "pinput-numeric",
         title: "pInput Numeric Demo",
       },
+      {
+        route: ["pinput-text"],
+        nav: true,
+        moduleId: PLATFORM.moduleName("./pInputTextDemo.html"),
+        name: "pinput-text",
+        title: "pInput Text Demo",
+      },
+      {
+        route: ["pform-item"],
```

Comment:
what if we call this "pform-input"?  makes more sense to me.  I'm not sure what "item" means in this context.

Reply:
This was inspired from one of the UI kits I used in the past. We can changes it, of course. What names do you suggest? 

What about "form-group", "input-group" or "form-field"?

Reply:
"pform-input"

Reply:
"pform-input"